Makefile

```diff
@@ -2,73 +2,26 @@
 # See the attached LICENSE file.
 # Copyright 2005, 2013 by Alain Frisch and LexiFi.
 
-include $(shell ocamlc -where)/Makefile.config
+INSTALL_ARGS := $(if $(PREFIX),--prefix $(PREFIX),)
 
-VERSION=1.99.2
-# Don't forget to change META file as well
+.PHONY: build install uninstall clean doc test all
 
-.PHONY: all opt clean test install package
-
-all:
-	(cd src/lib && $(MAKE) all doc)
-	(cd src/syntax && $(MAKE) all)
-
-opt:
-	(cd src/syntax && $(MAKE) opt)
-	(cd src/lib && $(MAKE) opt)
-
-clean:
-	rm -f *~ *.cm* *.a *.lib *.o *.obj
-	(cd src/lib && $(MAKE) clean)
-	(cd src/syntax && $(MAKE) clean)
-	(cd examples && $(MAKE) clean)
-	rm -rf libdoc
-
-test: clean all opt
-	cd examples && $(MAKE) test
-
-INSTALL=META src/syntax/sedlex.cma src/syntax/ppx_sedlex$(EXE) src/lib/sedlexing.cma src/lib/sedlexing.cmi
-
-INSTALL_OPT=src/syntax/sedlex.cmxs src/syntax/sedlex$(EXT_LIB) src/syntax/sedlex.cmxa src/syntax/ppx_sedlex.opt$(EXE) src/lib/sedlexing.cmx src/lib/sedlexing$(EXT_LIB) src/lib/sedlexing.cmxa
+build:
+	time -p jbuilder build @install
```

Comment:
Please remove the call to time, it's not useful.

Reply:
Done.